src/config/ConfigParser.ts

```diff
@@ -658,6 +795,67 @@ export function configParse29(slice: Slice | null | undefined) {
     throw new Error('Invalid config');
 }
 
+export function configParse31(slice: Slice | null | undefined) {
+    if (!slice) {
+        throw Error('Invalid config');
+    }
+
+    const rawAddrsDict = slice.loadDict(Dictionary.Keys.Buffer(32), Dictionary.Values.Uint(0))
+    
+    return [...rawAddrsDict].map(e => new Address(-1, e[0]))
+}
+
+export function configParse44(slice: Slice | null | undefined) {
+    if (!slice) {
+        throw Error('Invalid config');
+    }
+    const magic = slice.loadUint(8);
+    if (magic !== 0x00) {
+        throw new Error('Invalid config');
+    }
+
+    // buffer36 = uint288
+    const rawAddrsDict = slice.loadDict(Dictionary.Keys.Buffer(36), Dictionary.Values.Uint(0));
+    const suspendedUntil = slice.loadUintBig(32);
+    
+    // uint288 = [wc:int32 addr:uint256]
+    const constructedAddrs = [...rawAddrsDict].map(e => new Address(
+      e[0].readInt32BE(),
+      e[0].subarray(4),
+    ));
+
+    return {
+        addresses: constructedAddrs,
+        suspendedUntil
+    };
+}
+
+export function configParse45(slice: Slice | null | undefined) {
+    if (!slice) {
+        throw Error('Invalid config');
+    }
+    const magic = slice.loadUint(8);
+    if (magic !== 0xc0) {
+        throw new Error('Invalid config');
+    }
+
+    const precompiledContracts = slice.loadDict(Dictionary.Keys.Buffer(32), {
```

Comment:
Key and Value types should not be defined in-place. Please put them into the global scope.